src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewController.swift

```diff
@@ -26,12 +26,17 @@ class DataDonationViewController: DynamicTableViewController, DeltaOnboardingVie
 
 	override func viewDidLoad() {
 		super.viewDidLoad()
-		parent?.navigationItem.title = AppStrings.DataDonation.Info.title
-		parent?.navigationItem.largeTitleDisplayMode = .always
-		parent?.navigationController?.navigationBar.prefersLargeTitles = true
+		navigationItem.title = AppStrings.DataDonation.Info.title
```

Comment:
Would it be possible to have the 3 lines of configuration just in `viewWillAppear`?

Reply:
**navigation item:** all example code from apple I have seen so far perform the setup of navigation items in viewDidLoad, so I consider it as the "standard way" of implementation. But of course you can do it differently if you have reasons.
viewDidLoad is called once / at the beginning of the lifetime of the view controller. The navigation item is owned by the view controller, so there is no reason to configure it repeatedly in each 'viewWillAppear' call; a one-time configuration should be sufficient.

**navigation bar:** this is shared between the sibling view controllers. And to be honest, I am not too enthusiastic that this shared property is reconfigured by the view controllers all the time; I am currently investigating if I find a better solution (see #3372), but this investigation is still ongoing.

Note that the timing of 'viewDidLoad' and 'viewWillAppear' is slightly different, so there is a potential risk that moving these lines could have an impact on the animation. 
I tried to move all three lines into 'DataDonationViewController.viewWillAppear', and I saw no adverse effect; but I tested on simulator only, not on a true device.

Reply:
***I removed an inconsistency here and changed the behaviour from normal to large titles.
Please clarify with your UI responsibles if these large titles are ok.***
*(one side effect of the large titles is that the consent box is no longer visible without scrolling - and this might be undesired...)*

Reply:
@ndegendogo: Regarding separation of configuration in`viewDidLoad ` and `viewWillAppear`: keep it like that. Because only the configuration of the `navigationBar` can be reconfigured during a navigation to the next view controller. So it makes sense to only reconfigure `navigationBar` on reappearance.

Regarding the large titles: It would be very nice if you provide me screenshots of the different states. This way I can directly ask a designer about the change.

Reply:
First screenshot shows data donation screen as of current implementation (normal title).
Second screenshot shows same screen but with large title.
Both screenshot with iPhone SE (shorter display size).

![Simulator Screen Shot - iPhone SE (2nd generation) - 2021-09-01 at 10 54 22](https://user-images.githubusercontent.com/6118267/131652856-4b62644b-b311-44a0-a6ca-1cece6a3620b.png)

![Simulator Screen Shot - iPhone SE (2nd generation) - 2021-09-01 at 10 41 32](https://user-images.githubusercontent.com/6118267/131652893-605f8d57-dc9d-419b-beed-98a7358cafe0.png)

Reply:
@ndegendogo I had a chat with a responsible designer. Its a little more complicated. This view controller is used in the onboarding and also in the settings. The ideal solution would be: a small title in the onboarding and a large title in the settings.

Reply:
lol 😅😅

I'll try ...

Reply:
@ArturFriesen unfortunately, its a little more complicated .....

In the current branch release/2.10.x all onboarding screens have a large free space at the top - except the Data Donation, which has a large title. And ... yes, the Data Donation screen from Settings has a normal title.

*I hope your designer won't cry if he sees this ...*

I have submitted a ticket #3459 with screenshots.

My suggestion is now as follows:
1) confine the scope of this PR. I cannot save the world in one day. I'll role back all the changes related to the data donation screen, so at least I have no regression here.
2) fix the bug in the onboarding screens (separate PR). Maybe you guys have already seen this and started to work on the fix. If not, I could volunteer, or maybe one of you is faster.
3) still another PR: make your designer happy, as discussed above

Reply:
@ArturFriesen I have now reverted all changes releated to the Data Donation screen as suggested.

Additionally I found that this issue occurs also in the 'Background App Refresh' screen (from Settings).
I have fixed also this occurence.